bazel/repository_locations.bzl

```diff
@@ -68,6 +68,17 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         license = "Apache-2.0",
         license_url = "https://github.com/bazelbuild/rules_apple/blob/{version}/LICENSE",
     ),
+    com_github_bazelbuild_buildtools = dict(
+        project_name = "Bazel build tools",
+        project_desc = "Developer tools for working with Google's bazel buildtool.",
+        project_url = "https://github.com/bazelbuild/buildtools",
+        version = "5.1.0",
```

Comment:
cc @moderation this was pinned to this version on the basis that this is what we currently use an my initial testing suggested that a version change might create a flood of linting changes (not 100% but thought it better to separate that update - https://github.com/envoyproxy/envoy/pull/29405 - unfortunately github is borked atm so hard to test)

Reply:
ci is ~working again and confirmed that upgrading brings a load of change